docs/README.md

```diff
@@ -0,0 +1,256 @@
+# Microsoft Azure SDK for Ruby
+
+Microsoft Azure SDK for Ruby has been deprecated. It will be in maintenance mode for a period of time. This doc instructs how to use Azure REST API as a replacement. By the way, it is highly recommanded you to use GA SDKs
+* [Python](https://github.com/Azure/azure-sdk-for-python)
+* [Java](https://github.com/Azure/azure-sdk-for-java)
+* [JavaScript](https://github.com/Azure/azure-sdk-for-js)
+* [.Net](https://github.com/Azure/azure-sdk-for-net)
+
+You can also find other SDKs in [Azure SDK](https://github.com/Azure/azure-sdk).
+
+# Azure REST API
+
+[Azure REST API Reference](https://docs.microsoft.com/en-us/rest/api/azure/)
+
+
+## Authentication
+
+The first step to using the SDK is authentication and permissions. For people unfamilar with Azure this may be one of
```

Comment:
The first step to calling the REST API is authentication. For people who are unfamiliar with Azure, this maybe one of the challenging concepts. 

To authenticate, we will need to set up a Service Principal. For reference on how to set up a Service Principal from command line, please see [Authenticating a service principal with Azure Resource Manager](http://aka.ms/cli-service-principal) or [Unattended Authentication](http://aka.ms/auth-unattended).

For a more robust explanation of authentication in Azure, please follow  [Developer’s guide to auth with Azure Resource Manager API](http://aka.ms/arm-auth-dev-guide).

After creating the Service Principal, you should have the following key identifiers
- Client ID (GUID)
- Client Secret (String)
- Tenant ID (GUID) or domain name (String)

Reply:
Thanks Nick. It seemed that above link about Developer’s guide to auth with Azure Resource Manager API could not be opened.